tools/js-sdk-release-tools/src/test/changelog/changelogGenerator.test.ts

```diff
@@ -131,6 +131,46 @@ describe("Breaking change detection", () => {
             "Operation DataProductsCatalogs_sig_change.get has a new signature"
         );
     });
+
+    test("Patch RLC -> RLC's basic breaking changes", async () => {
+        const root = path.join(
+            __dirname,
+            "../../../packages/typescript-codegen-breaking-change-detector/misc/test-cases/patch-detection/"
+        );
+        const oldViewPath = path.join(__dirname, `testCases/patch.1.old.rlc.api.md`);
+        const newViewPath = path.join(__dirname, `testCases/patch.1.new.rlc.api.md`);
+        const changelog = await extractExportAndGenerateChangelog(
+            oldViewPath,
+            newViewPath,
+            SDKType.RestLevelClient,
+            SDKType.RestLevelClient
+        );
+        console.log("changelog --");
+        expect(changelog.displayChangeLog()).toBe(
+            `### Features Added
+
+  - Added operation in Routes for path: "add"
+  - Added Interface ClusterGetOld
+  - Added Type Alias typesAdd
+  - Added function funcAdd
+  - Added function overload "export function isUnexpected(response: C | E): response is C;"
+
+### Breaking Changes
+
+  - Removed operation Routes for path: "remove"
+  - Operation in "Routes" has a new signature for path: "change_return_type"
```

Comment:
The idea in RLC would ignore the interface name changes, if one operation in Routes only has return type name change, would it be deteced as a breaking?

Reply:
No, it wouldn't be a breaking change, and it is done in another merged PR